src/styles/variables.less

```diff
@@ -18,6 +18,21 @@
 @size-button-borderRadius: @size-borderRadius;
 
 // colors
+@color-white: #fff;
+@color-black: black;
```

Comment:
Why does `@color-white` use the hex `#fff` value while `@color-black` uses the color name `black`?